StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/ReadabilityRules/SA1130CodeFixProvider.cs

```diff
@@ -193,6 +201,14 @@ private static ImmutableArray<string> GetEqualsArgumentList(SemanticModel semant
             return namedTypeSymbol.DelegateInvokeMethod.Parameters.Select(ps => ps.Name).ToImmutableArray();
         }
 
+        private static ImmutableArray<string> GetMemberReturnTypeArgumentList(SemanticModel semanticModel, AnonymousMethodExpressionSyntax anonymousMethod)
+        {
+            var enclosingSymbol = semanticModel.GetEnclosingSymbol(anonymousMethod.Parent.SpanStart);
+            var returnType = (INamedTypeSymbol)((IMethodSymbol)enclosingSymbol).ReturnType;
```

Comment:
❓ Will the cast to `IMethodSymbol` fail for local functions?

```csharp
void Method() {
  EventHandler LocalFunction() => delegate { };
}
```

Reply:
It won't.